src/pages/VenueLandingPage/VenueLandingPage.tsx

```diff
@@ -52,9 +51,9 @@ import SecretPasswordForm from "components/molecules/SecretPasswordForm";
 import "./VenueLandingPage.scss";
 
 export interface VenueLandingPageProps {
-  venue: Firestore["data"]["currentVenue"];
+  venue?: Firestore["data"]["currentVenue"];
   venueEvents?: Firestore["ordered"]["venueEvents"];
-  venueRequestStatus: Firestore["status"]["requested"]["currentVenue"];
+  venueRequestStatus?: Firestore["status"]["requested"]["currentVenue"];
   purchaseHistory?: Firestore["ordered"]["userPurchaseHistory"];
   venueId?: string;
 }
```

Comment:
Why were these types changed here?

Reply:
Didn't allow for `<VenueLandingPage>` to be used without props, even though those props weren't being passed in through JSX code but retrieved inside the component.

Reply:
Is the type correct at all then? If we're not meant to be passing it props, those props should be removed from it's signature?

Reply:
This component is only used in `AppRouter` which doesn't pass any props to it. The component also never actually reads any props passed to it.

fixed by removing the type completely in f3ed798dfc0ba72a98457f68b714dbdb966a5ed7